Rubberduck.VBEEditor/ISourceCodeHandler.cs

```diff
@@ -0,0 +1,12 @@
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+
+namespace Rubberduck.VBEditor
+{
+    public interface ISourceCodeHandler
+    {
+        string Export(IVBComponent component);
+        void Import(IVBComponent component, string fileName);
+
+        string Read(IVBComponent component);
```

Comment:
I wonder if we really should be using `string` here. Would we be better off providing a `Stream` instead?

Reply:
Yep that would be better. Will change it when I get home from work.

Reply:
Hmm, so I had a look at the implementations of this interface. It creates a temp file, reads it all as text, then deletes it. I think this is the correct behaviour, tidying up as promptly as possible.

So I could take that data and expose it as a stream, but given that it's all in memory by that point anyway, I don't think it buys anything to do so.

Reply:
I'm not a huge fan of the name, but it'll have to do.

Reply:
Yeah I struggled with it... would Source**File**Handler be better?